web/src/components/ToggleButton.tsx

```diff
@@ -76,16 +74,16 @@ export default function ToggleButton({
                 role="button"
                 tabIndex={0}
                 className={twMerge(
-                  'b ml-2 h-6 w-6 select-none justify-center self-center rounded-full bg-white text-center drop-shadow dark:border dark:border-gray-500 dark:bg-gray-900',
+                  'b ml-2 inline-flex h-6 w-6 select-none   justify-center self-center rounded-full bg-white shadow dark:border dark:border-gray-500 dark:bg-gray-900',
                   isToolTipOpen && 'pointer-events-none'
                 )}
               >
-                <p>i</p>
+                <p className="size  self-center font-inter text-xs font-bold ">i</p>
```

Comment:
```suggestion
                <p className="size self-center font-inter text-sm font-bold">i</p>
```
Same here about the font sice